tests/test_server.py

```diff
@@ -241,249 +258,312 @@ async def test_sockets_not_started() -> None:
+    writer.write(b"?sensor-list[8] /(/\n")
+    await check_reply(reader, [re.compile(rb"^!sensor-list\[8\] fail .+\n$")])
 
 
 async def test_sensor_value(
-        mock_time, reader: asyncio.StreamReader, writer: asyncio.StreamWriter) -> None:
-    writer.write(b'?sensor-value[9]\n')
-    await check_reply(reader, [
-        b'#sensor-value[9] 123456789.0 1 auto-override unknown 0\n',
-        b'#sensor-value[9] 123456789.0 1 counter-queries nominal 0\n',
-        b'#sensor-value[9] 123456789.0 1 float-sensor unknown 0.0\n',
-        b'#sensor-value[9] 123456789.0 1 foo unknown first-value\n',
-        b'!sensor-value[9] ok 4\n'])
+    mock_time, reader: asyncio.StreamReader, writer: asyncio.StreamWriter
+) -> None:
+    writer.write(b"?sensor-value[9]\n")
+    await check_reply(
+        reader,
+        [
+            b"#sensor-value[9] 123456789.0 1 auto-override unknown 0\n",
+            b"#sensor-value[9] 123456789.0 1 counter-queries nominal 0\n",
+            b"#sensor-value[9] 123456789.0 1 float-sensor unknown 0.0\n",
+            b"#sensor-value[9] 123456789.0 1 foo unknown first-value\n",
+            b"!sensor-value[9] ok 4\n",
+        ],
+    )
 
 
 async def test_sensor_value_filter(
-        mock_time, reader: asyncio.StreamReader, writer: asyncio.StreamWriter) -> None:
+    mock_time, reader: asyncio.StreamReader, writer: asyncio.StreamWriter
+) -> None:
     # The other filter tests are omitted since they're covered by the
     # sensor-list tests.
-    writer.write(b'?sensor-value[9] counter-queries\n')
-    await check_reply(reader, [
-        b'#sensor-value[9] 123456789.0 1 counter-queries nominal 0\n',
-        b'!sensor-value[9] ok 1\n'])
+    writer.write(b"?sensor-value[9] counter-queries\n")
+    await check_reply(
+        reader,
+        [
+            b"#sensor-value[9] 123456789.0 1 counter-queries nominal 0\n",
+            b"!sensor-value[9] ok 1\n",
+        ],
+    )
 
 
 async def test_version_list(reader: asyncio.StreamReader, writer: asyncio.StreamWriter) -> None:
-    writer.write(b'?version-list[10]\n')
-    await get_version_info(reader, prefix=b'#version-list[10]')
-    assert await reader.readline() == b'!version-list[10] ok 3\n'
+    writer.write(b"?version-list[10]\n")
+    await get_version_info(reader, prefix=b"#version-list[10]")
+    assert await reader.readline() == b"!version-list[10] ok 3\n"
 
 
 async def test_halt(
-        server: DummyServer, reader: asyncio.StreamReader, writer: asyncio.StreamWriter) -> None:
-    writer.write(b'?halt[11]\n')
-    await check_reply(reader, [
-        b'!halt[11] ok\n',
-        br'#disconnect server\_shutting\_down' + b'\n',
-        b''])   # Empty string indicates EOF
+    server: DummyServer, reader: asyncio.StreamReader, writer: asyncio.StreamWriter
+) -> None:
+    writer.write(b"?halt[11]\n")
+    await check_reply(
+        reader, [b"!halt[11] ok\n", rb"#disconnect server\_shutting\_down" + b"\n", b""]
+    )  # Empty string indicates EOF
```

Comment:
This comment is no longer on the same line as the empty string. I think put the arguments as one per line again, with the comment on the line with the empty string.